purview/api/v1.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import pkg_resources
+
+from purview import app
+
+
+@app.route('/about')
+def version_info():
+    """
+    Extract version info from setup.py and display it on /about endpoint
+    :return: version info
+    """
+    try:
+        VERSION = pkg_resources.get_distribution('purview').version
+    except pkg_resources.DistributionNotFound:
+        VERSION = 'unknown (Please use setup.py to install purview)'
```

Comment:
Probably shouldn't expose this to the user in the event this occurs in prod some how :laughing:. How about just `unknown`?